package/index.js

```diff
@@ -129,6 +129,12 @@ function toTri(num) {
   return num.toString(3);
 }
 
+// changing number to a string
+
+function toStr(num) {
+return num.toString();
+}
+

```

Comment:
What's its use in a calculator?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
isnt it also about numbers?

Reply:
Here, you're converting a number to a string. What a user will do with that in terms of calculation?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
well... okay, edit it.